fossunited/chapters/doctype/foss_chapter_event_member/foss_chapter_event_member.py

```
@@ -18,8 +18,6 @@ class FOSSChapterEventMember(Document):
         parent: DF.Data
         parentfield: DF.Data
         parenttype: DF.Data
-        role: DF.Literal[
-            "Volunteer", "Core Team Member", "Graphic Designer", "Content Writer", "Marketing"  # noqa: F722, F821
-        ]
+        role: DF.Literal["Volunteer", "Core Team Member", "Graphic Designer", "Content Writer", "Marketing", "Speaker"]
```

Ruff (0.14.7)

Undefined name `Volunteer`

(F821)